frontend/src/app/modules/bookings/bookings-page/bookings-page.component.ts

```diff
@@ -80,12 +80,11 @@ export class BookingsPageComponent extends BaseComponent implements OnInit, OnDe
         const containerWidth = this.getPageSize();
 
         this.getNumberOfItemsToDisplay(containerWidth);
-        this.teamService.currentTeamEmitted$
-            .subscribe(teamId => {
-                this.teamId = teamId;
+        this.teamService.currentTeamEmitted$.subscribe((teamId) => {
+            this.teamId = teamId;
 
-                this.loadMeetings(this.getMeetingRequest());
```

Comment:
here, in subscribe, you have method that also has subscription. We have to avoid nested subscriptions. Use RxJs for this